### ISSUE TITLE
Support branding blockbook

### ISSUE THREAD

**Issue by youknowriad** [youknowriad/blockbook#4]
- Defining Colors.
 - A custom logo.
 - A cover page maybe.

**Comment by fabiankaegy** [comment]
@youknowriad  I played around with this a bit today and if we moved the `@import './reset.css';` before loading the blockbook we would already support overriding the css custom properties. 

This already makes it possible to theme a lot. Just the logo would need to be added :) Maybe a slot fill / filter could be cool.

![Screen Shot 2020-08-01 at 20 48 20](https://user-images.githubusercontent.com/20684594/89108391-58b90900-d438-11ea-8353-d68d30a3b4b8.png)

What do you think / have in mind for this?

**Comment by youknowriad** [comment]
yes, this could work but I feel we should offer a more friendly API (in addition to the CSS vars), maybe something like

```
import { setConfig } from 'blockbook-api';

setConfig({
  title: 'CoBlocks',
  color: 'red',
  url: ''
});
```

I think when we add this feature, we should also add something like "Powered by blockbook" somewhere.
